# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: widget/nsITouchBarHelper.idl
# Commit: 355b0329bd95
# Full Hash: 355b0329bd95d2a9573c04ece889c734da00f080
# Author: harry <htwyford@mozilla.com>
# Date: 2019-10-08 21:45:57
# Regressor Bug: 1563350
# File Overlap Count: 2
# Description:
#   Bug 1563350 - Add popover to the Touch Bar that displays when the Urlbar has focus. r=mikedeboer,spohl,fluent-reviewers,Pike,flod
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D38563
# ==============================================================================

diff -r 653caa0c494a -r 355b0329bd95 widget/nsITouchBarHelper.idl
--- a/widget/nsITouchBarHelper.idl	Tue Oct 08 00:52:07 2019 +0000
+++ b/widget/nsITouchBarHelper.idl	Tue Oct 08 00:52:07 2019 +0000
@@ -23,6 +23,11 @@
   readonly attribute AString activeTitle;
 
   /**
+   * Return true if the Urlbar has focus.
+   */
+  readonly attribute boolean isUrlbarFocused;
+
+  /**
    * Returns all available Touch Bar Inputs in an nsIArray
    * of nsITouchBarInput objects.
    */
